docs/metrics.md

```diff
@@ -138,6 +138,12 @@ The following metrics are added to the ping:
 | sync_auth.sign_in |[event](https://mozilla.github.io/glean/book/user/metrics/event.html) |A user pressed the sign in button on the sync authentication page and was successfully signed in to FxA  |[1](https://github.com/mozilla-mobile/fenix/pull/2745#issuecomment-494918532)||2020-03-01 |
 | sync_auth.sign_out |[event](https://mozilla.github.io/glean/book/user/metrics/event.html) |A user pressed the sign out button on the sync account page and was successfully signed out of FxA  |[1](https://github.com/mozilla-mobile/fenix/pull/2745#issuecomment-494918532)||2020-03-01 |
 | sync_auth.sign_up |[event](https://mozilla.github.io/glean/book/user/metrics/event.html) |User registered a new Firefox Account, and was signed into it  |[1](https://github.com/mozilla-mobile/fenix/pull/4931#issuecomment-529740300)||2020-03-01 |
+| tracking_protection.etp_setting_changed |[event](https://mozilla.github.io/glean/book/user/metrics/event.html) |A user added a TP exception through the TP toggle in the panel.  |[1](https://github.com/mozilla-mobile/fenix/pull/TBD)|<ul><li>strict: The new setting for ETP: etp_strict, etp_standard</li></ul>|2020-03-01 |
```

Comment:
Don't forget to update the "TBD" part of this with a data review 😄

Reply:
Oops just needed to rebuild and push :)